core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -1913,4 +1914,55 @@ public void testSqlForInvalidArguments() {
         .isInstanceOf(IllegalArgumentException.class)
         .hasMessage("Invalid dialect: (empty string)");
   }
+
+  @Test
+  public void dropNonEmptyNamespace() {
```

Comment:
Wouldn't it make sense to add this test case to other view catalogs (or at least to Nessie and Hive catalog)? Or maybe to a base class for those tests?